dataactbroker/handlers/fileHandler.py

```diff
@@ -1113,6 +1113,29 @@ def restart_validation(submission):
 
         return JsonResponse.create(StatusCode.OK, {"message": "Success"})
 
+    def fail_validation(self, upload_id):
+        # update all validation jobs to "ready"
+        sess = GlobalDB.db().session
+        initial_file_types = [FILE_TYPE_DICT['appropriations'], FILE_TYPE_DICT['program_activity'],
+                              FILE_TYPE_DICT['award_financial']]
+
+
+        jobs = sess.query(Job).filter(Job.submission_id == upload_id).all()
+
+        # set all jobs to their initial status of "waiting"
+        for job in jobs:
+            job.job_status_id = JOB_STATUS_DICT['waiting']
+
+        # update upload jobs to "running", only for files A, B, and C
```

Comment:
1) This is setting it to "failed" not "running"
2) Why are we setting them to "failed"? I think we can delete if it's in "waiting" can't we? Is this something I need to see the front end to understand?
3) assuming we need to set them to "failed" why do we have to set the other jobs to "waiting" at all? If the uploads are still running then the rest shouldn't be running? If we're trying to invalidate purely for deletion purposes, couldn't we just filter all running jobs and set them to failed? Or just set all jobs to "failed" since they're about to vanish anyway?
4) why do we need to filter `initial_file_types` out at all?